tt_metal/jit_build/build.cpp

```diff
@@ -64,6 +65,10 @@ void JitBuildEnv::init(uint32_t build_key, tt::ARCH arch)
     case ARCH::WORMHOLE_B0:
         common_flags = "-mwormhole -march=rv32imw -mtune=rvtt-b1 -mabi=ilp32 ";
         break;
+    case ARCH::BLACKHOLE:
+        // TODO (abhullar): Check with pgkeller on march value
+        common_flags = "-mblackhole -march=X -mtune=rvtt-b1 -mabi=ilp32 ";
```

Comment:
@pgkeller do you know what the -march should be?

Reply:
-march=rv32iml